.github/workflows/samples_configuration.yml

```diff
@@ -9,7 +9,7 @@ on:
     - cron: "40 22 * * *" # Every day starting at 6:40 BJT
   pull_request:
     branches: [ main ]
-    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml, '!examples/flows/integrations/**' ]
+    paths: [ examples/**, examples/*requirements.txt, .github/workflows/samples_configuration.yml ]
```

Comment:
please reverted such changes to exclude integration examples

Reply:
Created a bug to track. [Bug 2839343](https://msdata.visualstudio.com/Vienna/_workitems/edit/2839343): [Promptflow SDK& CLI] Fix readme CI template